pkgs/applications/audio/squeezelite/default.nix

```
@@ -26,7 +68,8 @@ stdenv.mkDerivation {
   meta = with stdenv.lib; {
     description = "Lightweight headless squeezebox client emulator";
     homepage = https://github.com/ralph-irving/squeezelite;
-    license = licenses.gpl3;
+    license = with licenses; [ gpl3 ] ++ optional dsdSupport [ bsd2 ];
+    maintainers = maintainers.samdoshi;
```

in case others want to join, make it easier for them
```suggestion
    maintainers = with maintainers; [ samdoshi ];
```